appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/page-components/reject.mapper.js

```diff
@@ -220,56 +220,6 @@ export function rejectCheckYourAnswersPage(appealDetails, comment, rejectionReas
 	return pageContent;
 }
 
-/**
- * @param {Representation} comment
- * @param {RepresentationRejectionReason[]} rejectionReasonOptions
- * @param {import('@pins/express').Session} session
- * @param {{ optionId: number, message: string }} [error]
- * @returns {import('../../../../../appeals/appeals.types.js').CheckboxItemParameter[]}
- */
-export function mapRejectionReasonOptionsToCheckboxItemParameters(
-	comment,
-	rejectionReasonOptions,
-	session,
-	error
-) {
-	const rejectionReasons = comment.rejectionReasons || [];
-	const rejectionReasonMap = new Map(rejectionReasons.map((reason) => [reason.id, reason]));
-
-	const selectedReasons = (() => {
-		const value = session.rejectIpComment?.rejectionReason;
```

Comment:
When I moved this I also added a `sessionKey` arg so this works for multiple rep types